bin/postman-test/collections/device-virtual.postman_collection.json

```diff
@@ -36,7 +36,7 @@
 							"});",
 							"",
 							"pm.test(\"Response body is correct\", function () {",
-							"    pm.expect(pm.response.text()).to.include(\"1.1.0\");",
+							"    pm.expect(pm.response.text().length>0).to.be.true; ",
```

Comment:
verify the status code is enough.  we can totally remove this response body checking.

Reply:
removed